src/main/java/pacApp/pacModel/CarType.java

```diff
@@ -0,0 +1,5 @@
+package pacApp.pacModel;
+
+public enum CarType {
+    SMALL, MEDIUM, LARGE
+}
```

Comment:
Wouldn't it make sense to set the costs multiplier for those types here?

Reply:
Absolutely, I have added an additional method in the CarType enum. Unfortunately, it seems that it is not possible to use fractional numbers directly for enum types. 
[Stackoverflow Link](https://stackoverflow.com/questions/14744123/can-you-use-enum-for-double-variables)